pkgs/development/python-modules/bc-python-hcl2/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, buildPythonPackage, fetchPypi, nose }:
+
+let
+  lark-parser = buildPythonPackage rec {
+    pname = "lark-parser";
+    version = "0.7.8";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "JiFeuxV+b7LudDGapERbnzt+RW4mviFc4Z/aqpAcIKQ=";
+    };
+
+    doCheck = true;
+  };
+in
+buildPythonPackage rec {
+  pname = "bc-python-hcl2";
+  version = "0.3.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "VZhI1oJ2EDZGyz3iI6/KYvJq4BGafzR+rcSgHqlUDrA=";
+  };
+
+  propagatedBuildInputs =  [
+    lark-parser
+    nose
```

Comment:
I this required at runtime?

Reply:
I will change this to `buildInputs`.

Sorry, I didn't carefully choose the dependencies' type.

Reply:
please use `checkInputs` for test only dependencies.